packages/hardhat-chai-matchers/src/internal/reverted/reverted.ts

```diff
@@ -1,15 +1,21 @@
+import { buildAssert } from "../../utils";
 import { decodeReturnData, getReturnDataFromError } from "./utils";
 
 export function supportReverted(Assertion: Chai.AssertionStatic) {
   Assertion.addProperty("reverted", function (this: any) {
+    // see buildAssert's jsdoc
+    const negated = this.__flags.negate;
```

Comment:
I'd like to see a one-line explanation here, something like `capture negated flag before async code executes. see buildAssert's jsdoc.`